### ISSUE TITLE
Display error messages to users if something goes wrong when deleting imagery

### ISSUE THREAD

**Issue by lambdatronic** [openforis/collect-earth-online#404]
There is currently a constraint in the database that prevents imagery from being deleted if samples have already been classified with it. However, we don't actually send this error back to the user on the Review Institution page. Instead, if you try to delete such an imagery entry from your institution, you just get a message that says (Imagery Deleted Successfully). However, it obviously is not deleted.

Please catch errors on the server side and send them back to the UI
for display. You can use the flash_messages attribute on the response
object to do this.

**Comment by rfontanarosa** [comment]
Mmmm, I've added just one line (res.status(500);) on the back-end to make it work.

I discovered that flash_message is essentially used by JsonUser and PostgresUser and it's based on passing the error message server side performing a redirect.

I've also noted that the institution page is very slow due the loading of all the projects (it makes a call for every listed project), so reloading the entire page only for an error couldn't be the best way. Better manage this front-end side.
